src/controllers/handlers/parcel-operators-handler.ts

```diff
@@ -0,0 +1,26 @@
+import { Parcel } from '@dcl/schemas'
+import { HandlerContextWithPath, InvalidRequestError } from '../../types'
+import { ParcelOperators } from '../../adapters/parcel-rights-fetcher'
+
+export async function parcelOperatorsHandler(
+  context: HandlerContextWithPath<'parcelRightsFetcher' | 'logs', '/parcels/:x/:y/operators'>
+): Promise<{ status: 200; body: ParcelOperators }> {
+  const { x, y } = context.params
+  const { parcelRightsFetcher } = context.components
+
+  const logger = context.components.logs.getLogger('parcel-operators-handler')
+  const xInt = parseInt(x)
+  const yInt = parseInt(y)
+
+  if (!Parcel.validate({ x: xInt, y: yInt })) {
+    logger.error(`Invalid values for coordinates: x=${x}, y=${y}`)
```

Comment:
Would you mind switching the error to an info? It would be great to log errors only when there's an error on our code:
```suggestion
    logger.info(`Invalid values for coordinates: x=${x}, y=${y}`)
```